homeassistant/components/aprilaire/select.py

```diff
@@ -0,0 +1,157 @@
+"""The Aprilaire select component."""
+
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import cast
+
+from pyaprilaire.const import Attribute
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AprilaireCoordinator
+from .entity import BaseAprilaireEntity
+
+AIR_CLEANING_EVENT_MAP = {0: "off", 3: "event_clean", 4: "allergies"}
+AIR_CLEANING_MODE_MAP = {0: "off", 1: "constant_clean", 2: "automatic"}
+FRESH_AIR_EVENT_MAP = {0: "off", 2: "3hour", 3: "24hour"}
+FRESH_AIR_MODE_MAP = {0: "off", 1: "automatic"}
```

Comment:
Maybe just create the inverse maps as a constant outside the class as well

Reply:
That would require adding a new parameter as well. Is there any real benefit to having the inverse maps as constants that then get passed in, as opposed to just generating the inverse map in the init?